include/rangers-api/ApplicationCommon/FSM/GOCHsm2.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace app_cmn::fsm {
+    class GOCHsm2 : public hh::game::GOComponent {
+        uint64_t unk101;
+        hh::ut::HsmBase hsm;
+        int64_t unk102;
+        bool unk103;
+        uint32_t unk104;
+        csl::ut::InplaceMoveArray<void*, 1> unk105;
+        uint64_t unk106;
+        uint64_t unk107;
+    public:
+        GOCHsm2(csl::fnd::IAllocator* pAllocator);
+        static GOCHsm2* Instantiate(csl::fnd::IAllocator* pAllocator);
```

Comment:
Instantiate function name is never used in these games, these functions are more representative of Create functions in [LWAPI](https://github.com/hedge-dev/LWAPI/blob/9affbbfc3d30b81ca22bb8c76e6e13557e876d78/Sonic2013/Hedgehog/Game/GOComponent.h#L83).